style.css

```diff
@@ -420,3 +420,104 @@ ul {
   color: #e2505c;
   border: 1px solid #e2505c;
 }
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./images/image1.png);
+  background-repeat: no-repeat;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 24%;
+}
+
+form > input {
+  height: 52px;
+  padding-top: 4px;
+  padding-bottom: 33px;
+  padding-left: 16px;
+  border-radius: 8px;
+}
+
+.form-section > p {
+  width: 25%;
+  height: 64px;
+  left: 16px;
+  margin-top: 1px;
+}
+
+.form-section > h1 {
+  width: 100%;
+  height: 50px;
+  left: 16px;
+  font-family: "Playfair Display", sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  text-align: left;
+  padding: 1px;
+  margin-top: 106px;
+}
+
+#cbn-b {
+  width: 100%;
+  height: 43px;
+  left: 40px;
+  top: 4652px;
+  background: #1a2236;
+  color: #fff;
+  margin-bottom: 40px;
+}
```

Comment:
- [ ] **[OPTIONAL]** We could use `cursor: pointer;` as the last code reviewer suggested to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) 🙏
- [ ] **[OPTIONAL]** Please 🙏🏽 implement the button interactions (enable, hover, etc.).

Reply:
@franclobo Done...do i have to resubmit it again for review or you can check it from here?